andela/ah-frontend-thanos
# 161348754 Users should be able to presented with a reset pa…

#### What does this PR do?
Adds feature for resetting a password

#### Description of Task to be completed?
- Get the user email that requires a password reset
- Add email input form
- Add a password form with 2 input field (enter a new password and confirm password)
#### How should this be manually tested?
- Go to the password reset URL: `/passwordreset` 
- You should be able to enter your email from which you will receive an email on successful submission
- Click on the link sent to your email to reset the password

#### Any background context you want to provide?
Only users with existing accounts can reset password

#### What are the relevant pivotal tracker stories?
[#161348754](https://www.pivotaltracker.com/story/show/161348754)

#### Screenshots (if appropriate)
Password reset page
![image](https://user-images.githubusercontent.com/39625835/49859904-b9ca3e00-fe09-11e8-931d-4add3aed399d.png)

Redirect from the email sent
![image](https://user-images.githubusercontent.com/39625835/49860178-5391eb00-fe0a-11e8-9cad-f90c6d2be5bd.png)
